data/sea/52-psv.h

```diff
@@ -435,7 +435,9 @@ static ierror_loc_t psv_read_whole_buffer (psv_config_t *cfg, psv_state_t *s)
                 error = psv_write_dropped_entity_cur (s, error);
                 if (error) {
                     const char *error_pretty = ierror_loc_pretty (error, line);
-                    const char *error_msg    = strncat ("failed to drop entity:\n", error_pretty, error_msg_size);
+                    const int   msg_size     = error_msg_size + 32;
+                    char        error_msg[msg_size];
+                    snprintf (error_msg, msg_size, "failed to drop entity:\n %s", error_pretty);
```

Comment:
I reckon don't bother adding `32`, `error_msg_size` should be large enough already.